### PR TITLE
Add config to keep initial noise during validation

### PR DESCRIPTION
**Pull request by congruency**
I found it was useful to reduce the amount of moving parts when testing different learning rates. Seeing how the image changed with a fixed "seed" seemed like the most intuitive way of accomplishing that.

### CODE REVIEW

**Comment by congruency** [comment]
An example validating every 25 steps:
https://github.com/PixArt-alpha/PixArt-sigma/assets/6956712/fe4fc29a-b83a-42a9-bcf6-f95a7d2660b1

File: train_scripts/train.py [hunk 1]
```diff
@@ -54,9 +54,12 @@ def log_validation(model, step, device, vae=None):
     logger.info("Running validation... ")
     image_logs = []
     latents = []
-
+    
     for prompt in validation_prompts:
-        z = torch.randn(1, 4, latent_size, latent_size, device=device)
+        if noise:
```
Comment (GavChap) [inline train_scripts/train.py:59]: Shouldn't this be `if noise is not None` ?

Comment (bitaffinity) [reply train_scripts/train.py:59]: I agree, thank you for the review

Comment (congruency) [reply train_scripts/train.py:59]: I agree, thank you for the review

**Comment by lawrence-cj** [comment]
Good one! Merged!

**Pull request merged**